backend/src/models/studentApplySteps.js

```diff
@@ -0,0 +1,26 @@
+const mongoose = require('mongoose')
+
+const studApplyStepsSchema = mongoose.Schema({
+    number: {
+        type: String,
+        required: true
+    },
+    title:
+    {
+        type: String,
+        required: true
+    },
+    body:
+    {
+        type: String,
+        required: true
+    },
+    divider: {
+        type: String,
+        required: false
+
+    }
```

Comment:
Since divider is constant, I think we don't need to store it in database. We'll just define it the frontend app

Reply:
done